packages/fxa-admin-panel/package.json

```diff
@@ -75,7 +75,7 @@
     "fxa-shared": "workspace:*",
     "jest": "27.5.1",
     "jest-watch-typeahead": "0.6.5",
-    "nx": "21.1.2",
+    "nx": "21.2.4",
```

Comment:
Can't we just remove? (And other references to nx in package workspaces)?

Reply:
YES!  I had removed these but added them back when I encountered unrelated db errors.  Thanks for the reminder, I was excited for these to go!